### ISSUE TITLE
Add a child work accessibility audit 

### ISSUE THREAD

**Issue by kellyChex** [samvera/hyrax#4004]
### Descriptive summary

QA Testing for Hyrax
Hyrax Version: 3.0.0.pre.rc1
Hyrax Branch: master
Hyrax Revision: 9b2360f
© 2019 Samvera-Hyrax
Nurax Version
ENV: production
SHA: aa07ac6ed440e9f700cce91fc602c115482e6bed
Branch: 2019-09-16-10-00
Updated: 16 Sep 2019 15:00:28

In testing for the Hyrax 3.0 release, the following points are actions that were discovered during a FireFox tab test and axe accessibility audit. Note, only certain issues are being reported at this time.

**Elements must have sufficient color contrast**
![Screen Shot 2019-09-16 at 12 42 24 PM](https://user-images.githubusercontent.com/9621070/64988058-75c80600-d87f-11e9-8717-bc8c286f191f.png)

![Screen Shot 2019-09-16 at 12 42 53 PM](https://user-images.githubusercontent.com/9621070/64988087-85474f00-d87f-11e9-8ab6-aa8e9d595f72.png)

**id attribute value must be unique**

![Screen Shot 2019-09-16 at 12 43 26 PM](https://user-images.githubusercontent.com/9621070/64988129-98f2b580-d87f-11e9-99d4-62d3503582b3.png)

**ARIA role must be appropriate for the element**

![Screen Shot 2019-09-16 at 12 43 56 PM](https://user-images.githubusercontent.com/9621070/64988148-ac058580-d87f-11e9-8ef1-d227289397bb.png)

![Screen Shot 2019-09-16 at 12 44 16 PM](https://user-images.githubusercontent.com/9621070/64988171-b758b100-d87f-11e9-9e1f-a9c4bcbf8776.png)

![Screen Shot 2019-09-16 at 12 44 36 PM](https://user-images.githubusercontent.com/9621070/64988193-c3dd0980-d87f-11e9-9127-340ffc2cd570.png)

**Aside must not be contained in another landmark**
![Screen Shot 2019-09-16 at 12 45 35 PM](https://user-images.githubusercontent.com/9621070/64988246-e7a04f80-d87f-11e9-8890-0824725b35b3.png)

![Screen Shot 2019-09-16 at 12 45 56 PM](https://user-images.githubusercontent.com/9621070/64988265-f2f37b00-d87f-11e9-8bf4-913cdd44a1cf.png)

**Main landmark must not be contained in another landmark**
![Screen Shot 2019-09-16 at 12 46 31 PM](https://user-images.githubusercontent.com/9621070/64988312-0868a500-d880-11e9-8cdd-2ec602a0e6a1.png)

**Document must have one main landmark**

![Screen Shot 2019-09-16 at 12 47 14 PM](https://user-images.githubusercontent.com/9621070/64988355-22a28300-d880-11e9-9c29-35c37e82305a.png)

**Ensures landmarks are unique**
![Screen Shot 2019-09-16 at 12 48 02 PM](https://user-images.githubusercontent.com/9621070/64988396-3d74f780-d880-11e9-9779-bdb27f296d3f.png)

![Screen Shot 2019-09-16 at 12 48 21 PM](https://user-images.githubusercontent.com/9621070/64988423-49f95000-d880-11e9-9ebd-06558895e39c.png)

![Screen Shot 2019-09-16 at 12 48 39 PM](https://user-images.githubusercontent.com/9621070/64988451-541b4e80-d880-11e9-8d5a-7e6c9f06be41.png)

**Radio inputs with the same name attribute value must be part of a group**
![Screen Shot 2019-09-16 at 12 49 18 PM](https://user-images.githubusercontent.com/9621070/64988503-6eedc300-d880-11e9-9c85-2b0292845523.png)

### Steps to reproduce the behavior

1. Install the Axe plugin or addon from Deque (https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/ is the firefox version).
2. Go to Relationships tab of existing work and deposit new work as a child of this work 
3. Open the inspector (right-click, and select "Inspect element")
4. Click on the 'Axe' tab
5. Click 'Analyze'

### Tab Test 

In Firefox and Safari, unable to tab to the Deposit new work as child of this work button. Unable to access other buttons as well. 

Chrome works well. 

![Screen Recording 2019-09-16 at 12 40 PM](https://user-images.githubusercontent.com/9621070/64987977-41544a00-d87f-11e9-88af-5b60aa23044d.gif)
